Source/Core/Common/CommonPaths.h

```diff
@@ -27,89 +26,26 @@
 #define DOLPHIN_DATA_DIR "dolphin-emu"
 #endif
 
-// Shared data dirs (Sys and shared User for Linux)
-#if defined(_WIN32) || defined(LINUX_LOCAL_DEV)
-#define SYSDATA_DIR "Sys"
-#elif defined __APPLE__
-#define SYSDATA_DIR "Contents/Resources/Sys"
-#elif defined ANDROID
-#define SYSDATA_DIR "/sdcard/dolphin-emu"
-#else
-#ifdef DATA_DIR
-#define SYSDATA_DIR DATA_DIR "sys"
-#else
-#define SYSDATA_DIR "sys"
-#endif
-#endif
-
-// Dirs in both User and Sys
-#define EUR_DIR "EUR"
-#define USA_DIR "USA"
-#define JAP_DIR "JAP"
-
-// Subdirs in the User dir returned by GetUserPath(D_USER_IDX)
-#define GC_USER_DIR "GC"
-#define WII_USER_DIR "Wii"
-#define CONFIG_DIR "Config"
+// In the User dir
+#define ANAGLYPH_DIR "Anaglyph"
 #define GAMESETTINGS_DIR "GameSettings"
 #define MAPS_DIR "Maps"
-#define CACHE_DIR "Cache"
-#define SHADERCACHE_DIR "Shaders"
-#define STATESAVES_DIR "StateSaves"
-#define SCREENSHOTS_DIR "ScreenShots"
-#define LOAD_DIR "Load"
-#define HIRES_TEXTURES_DIR "Textures"
-#define DUMP_DIR "Dump"
-#define DUMP_TEXTURES_DIR "Textures"
-#define DUMP_FRAMES_DIR "Frames"
-#define DUMP_AUDIO_DIR "Audio"
-#define DUMP_DSP_DIR "DSP"
-#define DUMP_SSL_DIR "SSL"
-#define LOGS_DIR "Logs"
-#define MAIL_LOGS_DIR "Mail"
+#define RESOURCES_DIR "Resources"
 #define SHADERS_DIR "Shaders"
+#define THEMES_DIR "Themes"
+#define WII_USER_DIR "Wii"
+#define WII_SYSCONF "SYSCONF"
 #define WII_SYSCONF_DIR "shared2" DIR_SEP "sys"
 #define WII_WC24CONF_DIR "shared2" DIR_SEP "wc24"
-#define RESOURCES_DIR "Resources"
-#define THEMES_DIR "Themes"
-#define ANAGLYPH_DIR "Anaglyph"
-#define PIPES_DIR "Pipes"
-#define MEMORYWATCHER_DIR "MemoryWatcher"
-#define WFSROOT_DIR "WFS"
-#define BACKUP_DIR "Backup"
 
-// This one is only used to remove it if it was present
-#define SHADERCACHE_LEGACY_DIR "ShaderCache"
+// Dirs in both User and Sys
+#define EUR_DIR "EUR"
+#define USA_DIR "USA"
+#define JAP_DIR "JAP"
 
 // The theme directory used by default
 #define DEFAULT_THEME_DIR "Clean"
 
-// Filenames
-// Files in the directory returned by GetUserPath(D_CONFIG_IDX)
-#define DOLPHIN_CONFIG "Dolphin.ini"
-#define GCPAD_CONFIG "GCPadNew.ini"
-#define WIIPAD_CONFIG "WiimoteNew.ini"
-#define GCKEYBOARD_CONFIG "GCKeyNew.ini"
-#define GFX_CONFIG "GFX.ini"
-#define DEBUGGER_CONFIG "Debugger.ini"
-#define LOGGER_CONFIG "Logger.ini"
-#define UI_CONFIG "UI.ini"
-
-// Files in the directory returned by GetUserPath(D_LOGS_IDX)
-#define MAIN_LOG "dolphin.log"
-
-// Files in the directory returned by GetUserPath(D_WIISYSCONF_IDX)
-#define WII_SYSCONF "SYSCONF"
-
-// Files in the directory returned by GetUserPath(D_DUMP_IDX)
-#define RAM_DUMP "ram.raw"
-#define ARAM_DUMP "aram.raw"
-#define FAKEVMEM_DUMP "fakevmem.raw"
-
-// Files in the directory returned by GetUserPath(D_MEMORYWATCHER_IDX)
-#define MEMORYWATCHER_LOCATIONS "Locations.txt"
-#define MEMORYWATCHER_SOCKET "MemoryWatcher"
-
 // Sys files
 #define TOTALDB "totaldb.dsy"
```

Comment:
I looked at it, many of these are used in adjacent string literals, so it'd be a really large change. Probably an improvement for a future time.